### ISSUE TITLE
Association of the Cervical Microbiota With Pregnancy Outcome in a Subfertile Population Undergoing In Vitro Fertilization: A Case-Control Study

### ISSUE THREAD

**Issue by Rahila-me** [waldronlab/BugSigDBcuration#346]
Association of the Cervical Microbiota With Pregnancy Outcome in a Subfertile Population Undergoing In Vitro Fertilization: A Case-Control Study - Hao X, - frontiers
https://www.frontiersin.org/articles/10.3389/fcimb.2021.654202/full

**Comment by SvetlanaUP** [comment]
@Rahila-me please see my message here https://github.com/waldronlab/BugSigDBcuration/issues/338

You could check if reviewed issues are corrected in bugsigdb, if not claim it for correction and/or collaborate with the main curator.

**Comment by SvetlanaUP** [comment]
Noted as a self-assigned curation.

**Comment by Rahila-me** [comment]
@SvetlanaUP Good morning, I have gone through the whole github issues, looking for the one not marked needs review, last week found none, others (few) not marked people and the own is already collaborating . So last Thursday weekly meeting I raised the issue up that my work has been reviewed and i did well currently i don't have anything i am working, @cmirzayi says we can create issues then curate them. That's why.  So should i stop. Waiting for your kind reply thank you.

**Comment by Rahila-me** [comment]
@SvetlanaUP At this (@Rahila-me please see my message here https://github.com/waldronlab/BugSigDBcuration/issues/338

You could check if reviewed issues are corrected in bugsigdb, if not claim it for correction and/or collaborate with the main curator.)
You mean i should checkout reviewed issues, and update the correction based on the comment our mentors left there?, If so i will gladly do that also.

**Comment by SvetlanaUP** [comment]
@Rahila-me could you please help out here in correcting https://github.com/waldronlab/BugSigDBcuration/issues/155

**Comment by Rahila-me** [comment]
@SvetlanaUP Ok, On it.

**Comment by Scholarpat** [comment]
Hello @Rahila-me,

This is a wonderful curation, well done. I've made a few changes:

1. **Condition**: Changed from "female infertility" to "pregnancy." Although infertile females were recruited, the study aimed to characterize the cervical microbiota in pregnant females vs. non-pregnant females undergoing IVF.

2. **Statistical Test**: Both LEfSe and Wilcoxon tests were initially curated and both were used to detect significantly different abundances as reported in the study. However, the curated signatures were presented as a LEfSe analysis chart. Therefore, I've excluded Wilcoxon from this field.

3. **Signatures**: Changed Leuconostocaceae to its heterotypic synonym, Lactobacillaceae, and also added missing taxa.

This curation is now completed @SvetlanaUP. 

Thank you!
